tests/unit/DebugUtilsTest.ts

```diff
@@ -0,0 +1,546 @@
+import DebugUtils from '@libs/DebugUtils';
+import CONST from '@src/CONST';
+import type {Report, ReportAction} from '@src/types/onyx';
+
+const MOCK_REPORT: Report = {
+    reportName: 'Chat Report',
+    isOptimisticReport: false,
+    type: 'chat',
+    isOwnPolicyExpenseChat: false,
+    isPinned: false,
+    lastActorAccountID: 2,
+    lastMessageTranslationKey: '',
+    lastMessageHtml: '',
+    lastReadTime: '2024-08-08 18:20:44.175',
+    lastVisibleActionCreated: '2024-08-08 18:20:44.171',
+    notificationPreference: 'hidden',
+    oldPolicyName: '',
+    ownerAccountID: 0,
+    participants: {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        '14365522': {
+            hidden: true,
+            role: 'member',
+        },
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        '15083345': {
+            hidden: true,
+            role: 'admin',
+        },
+    },
+    policyID: '_FAKE_',
+    reportID: '2305929876534021',
+    stateNum: 0,
+    statusNum: 0,
+    description: '',
+    pendingFields: {},
+    chatType: CONST.REPORT.CHAT_TYPE.GROUP,
+    currency: 'USD',
+    errorFields: {},
+    hasOutstandingChildRequest: false,
+    hasOutstandingChildTask: false,
+    isCancelledIOU: false,
+    isWaitingOnBankAccount: false,
+    lastActionType: 'CREATED',
+    lastMessageText: '',
+    lastReadSequenceNumber: 0,
+    lastVisibleActionLastModified: '2024-08-08 18:20:44.171',
+    managerID: 0,
+    nonReimbursableTotal: 0,
+    permissions: ['read', 'write'],
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    private_isArchived: '',
+    total: 0,
+    unheldTotal: 0,
+    writeCapability: 'all',
+} satisfies Report;
+
+const MOCK_REPORT_ACTION: ReportAction = {
```

Comment:
Is possible to use [createRandomReportAction()](https://github.com/Expensify/App/blob/5a2602c441170093e99850e2194808218b0aa838/tests/utils/collections/reportActions.ts) here?